helmfile/values/falcosidekick.yaml.gotmpl

```diff
@@ -1,24 +0,0 @@
-config:
```

Comment:
Should this file be removed?

Reply:
It is removed :slightly_smiling_face:

Reply:
Because it's no longer needed or?

Reply:
It's now installed through the subchart included with falco, so we don't install it directly.
The old falcosidekick chart should also be removed from `charts/`.

Reply:
Okay, then it makes sense.

Reply:
Sorry, misunderstood your question. 

@aarnq I was thinking that the folder can be removed at the same time as https://github.com/elastisys/compliantkubernetes-apps/blob/b3c601e94de5e95ded0da6d2a8c2e0524f829c3c/helmfile/50-applications.yaml#L470 after the next release. But let me know if you would rather have me remove the folder now.

Reply:
@viktor-f The folder might as well be deleted now along with the values file. As long as `installed: false` is set then it won't check if either exists.